### ISSUE TITLE
re-usable css/js framework for admin interface

### ISSUE THREAD

**Issue by gerrit** [radiant/radiant#139]
I've recently had to adjust the admin interface of a few extensions to fit the new admin interface from 0.9.

This turned out to be quite repetetive and hacky. One extension e.g. needed a tab-control similar to the main page edit area. Because the CSS & JS that comes with radiant is overly specific (uses ID selectors for instance) I had to basically copy&paste the entire JS&CSS for this functionality. Of course, next time Radiant makes even a tiny colour change I'll have to adjust. Other examples of this are tables & floating popups.

It would be great if Radiant had some sort of documented html/css/js framework that extensions can use so they fit in to radiant automatically. Apart from ensuring visual consistency, this would make writing admin interfaces for extensions a whole lot easier and faster.

Ideally there would also be helper methods to generate the appropriate markup. The javascript hooks can probably all be done unobtrusively.

Basically, just changing the CSS a little bit would already be a huge benefit for extension authors.

If anyone else thinks this is a good idea I'm quite willing to have a go at this.
